code/game/objects/items/weapons/tanks/tank_types.dm

```diff
@@ -49,11 +48,12 @@
 /obj/item/weapon/tank/air
 	name = "air tank"
 	desc = "Mixed anyone?"
-	icon_state = "oxygen"
+	icon_state = OXYGEN

```

Comment:
inappropriate replacement, this needs to be ```"oxygen"``` (no way to avoid hardcoding it since it has to match the external .dmi file)